videos-list/03-viola.md

```diff
@@ -13,12 +13,41 @@ Discourse Discussion
 https://discourse.pymc.io/t/keynote-inferring-the-spread-of-sars-cov-2-and-measures-to-mitigate-it-by-viola-priesemann/6179
 
 ## Timestamps
-- 0:00 Start of event
-- x:xx 
-- x:xx
-
-## Note: help us add timestamps here
-https://github.com/pymc-devs/video-timestamps
+00:00:00 Introductory words by Viola
+00:00:48 The daily new confirmed COVID-19 cases
+00:01:39 The agenda of the talk
```

Comment:
```suggestion
00:00 Introductory words by Viola
00:48 The daily new confirmed COVID-19 cases
01:39 The agenda of the talk
```

Reply:
@bsenst 
For videos that are less than 1 hour long, we do not need the first two "00:".  Can we remove here and also for the rest of the timestamps in this file?  Thank you.